diff -Nru a/pcap-config.in b/pcap-config.in
--- a/pcap-config.in	2018-06-24 19:54:20.000000000 +0200
+++ b/pcap-config.in	2018-08-16 07:10:09.254672902 +0200
@@ -41,16 +41,6 @@
 	esac
 	shift
 done
-if [ "$V_RPATH_OPT" != "" ]
-then
-	#
-	# If libdir isn't /usr/lib, add it to the run-time linker path.
-	#
-	if [ "$libdir" != "/usr/lib" ]
-	then
-		RPATH=$V_RPATH_OPT$libdir
-	fi
-fi
 if [ "$static" = 1 ]
 then
 	#
@@ -59,16 +49,16 @@
 	#
 	if [ "$show_cflags" = 1 -a "$show_libs" = 1 ]
 	then
-		echo "-I$includedir -L$libdir -lpcap $LIBS"
+		echo "-lpcap @LIBS@"
 	elif [ "$show_cflags" = 1 -a "$show_additional_libs" = 1 ]
 	then
-		echo "-I$includedir -L$libdir $LIBS"
+		echo "@LIBS@"
 	elif [ "$show_cflags" = 1 ]
 	then
-		echo "-I$includedir"
+		echo ""
 	elif [ "$show_libs" = 1 ]
 	then
-		echo "-L$libdir -lpcap $LIBS"
+		echo "-lpcap @LIBS@"
 	elif [ "$show_additional_libs" = 1 ]
 	then
 		echo "$LIBS"
@@ -80,15 +70,15 @@
 	#
 	if [ "$show_cflags" = 1 -a "$show_libs" = 1 ]
 	then
-		echo "-I$includedir -L$libdir $RPATH -l$PACKAGE_NAME"
+		echo "-lpcap"
 	elif [ "$show_cflags" = 1 -a "$show_additional_libs" = 1 ]
 	then
-		echo "-I$includedir"
+		echo ""
 	elif [ "$show_cflags" = 1 ]
 	then
-		echo "-I$includedir"
+		echo ""
 	elif [ "$show_libs" = 1 ]
 	then
-		echo "-L$libdir $RPATH -l$PACKAGE_NAME"
+		echo "-lpcap"
 	fi
 fi
