diff -up libpcap-1.10.0/configure.ac.2~ libpcap-1.10.0/configure.ac
--- libpcap-1.10.0/configure.ac.2~	2021-01-03 22:59:42.915006953 +0100
+++ libpcap-1.10.0/configure.ac	2021-01-03 23:05:12.201374327 +0100
@@ -895,7 +895,7 @@ linux)
 				;;
 
 			*)
-				if test -d $withval; then
+				if test -d $withval -a $withval != /usr; then
 					libnldir=-L${withval}/lib
 					incdir=-I${withval}/include
 				fi
